pageserver/src/tenant/timeline.rs

```diff
@@ -1451,9 +1450,16 @@ impl Timeline {
                     );
 
                     // The flush loop will update remote consistent LSN as well as disk consistent LSN.
-                    self.flush_frozen_layers_and_wait(last_record_lsn)
-                        .await
-                        .ok();
```

Comment:
For this it doesn't matter that there is nothing to freeze, we will still signal and then we opt-in to waiting.